frontend/app/services/MemberService.scala

```diff
@@ -126,8 +127,10 @@ class MemberService(identityService: IdentityService,
     def createPaidZuoraSubscription(sfContact: ContactId, stripeCustomer: Customer, paid: PaidMemberJoinForm): Future[String] =
       (for {
         zuoraSub <- createPaidSubscription(sfContact, paid, paid.name, paid.tier, stripeCustomer, campaignCode)
-      } yield zuoraSub.subscriptionName).andThen { case Failure(e) =>
-        logger.error(s"Could not create paid Zuora subscription for user ${user.id}", e)}
+      } yield zuoraSub.subscriptionName).andThen {
```

Comment:
minor nitpick is since it's only failure it might be clearer as onError instead of andThen